docs/redirects.yaml

```diff
@@ -0,0 +1,35 @@
+"en/latest/":
```

Comment:
So I wonder about this... Currently `docs.clusterhq.com/en/latest/` immediately redirects to `docs.clusterhq.com/en/1.9.0`. That means external people are not going to be linking to it, mostly, but rather to whatever version of docs they randomly found. And new visitors will get warning and that's OK I guess. So is this mostly for internal links from our marketing material/website?